crates/sozo/src/commands/options/mod.rs

```diff
@@ -1,10 +1,111 @@
-use scarb::core::Workspace;
-use toml::Value;
+use scarb::core::{ManifestMetadata, Workspace};
+use serde::Deserialize;
 
 pub mod account;
 pub mod starknet;
 pub mod world;
 
-pub(super) fn dojo_metadata_from_workspace(ws: &Workspace<'_>) -> Option<Value> {
-    ws.current_package().ok()?.manifest.metadata.tool_metadata.as_ref()?.get("dojo").cloned()
+pub(crate) fn dojo_metadata_from_workspace(ws: &Workspace<'_>) -> Option<DojoMetadata> {
+    Some(ws.current_package().ok()?.manifest.metadata.dojo())
+}
+
+#[derive(Default, Deserialize, Debug, Clone)]
+pub(crate) struct DojoMetadata {
+    env: Option<Environment>,
+}
+
+#[derive(Default, Deserialize, Clone, Debug)]
+pub struct Environment {
+    rpc_url: Option<String>,
+    account_address: Option<String>,
+    private_key: Option<String>,
+    keystore_path: Option<String>,
+    keystore_password: Option<String>,
+    world_address: Option<String>,
+}
+
+impl Environment {
+    pub fn world_address(&self) -> Option<&str> {
+        self.world_address.as_deref()
+    }
+
+    pub fn rpc_url(&self) -> Option<&str> {
+        self.rpc_url.as_deref()
+    }
+
+    pub fn account_address(&self) -> Option<&str> {
+        self.account_address.as_deref()
+    }
+
+    pub fn private_key(&self) -> Option<&str> {
+        self.private_key.as_deref()
+    }
+
+    #[allow(dead_code)]
+    pub fn keystore_path(&self) -> Option<&str> {
+        self.keystore_path.as_deref()
+    }
```

Comment:
this is currently not used anywhere, but the variable was presenting in `commands::*`